public/app/plugins/datasource/azuremonitor/dashboards/postgresFlexibleServer.json

```diff
@@ -1,8487 +1,7668 @@
-        "type": "text"
+      "transparent": true,
+      "type": "text"
+    },
+    {
+      "datasource": {
+        "type": "grafana-azure-monitor-datasource",
+        "uid": "${ds}"
       },
-      {
-        "datasource": {
-          "type": "grafana-azure-monitor-datasource",
-          "uid": "${ds}"
-        },
-        "description": "Indicates if the database is up or not",
-        "fieldConfig": {
-          "defaults": {
-            "mappings": [
-              {
-                "options": {
-                  "0": {
-                    "color": "red",
-                    "index": 1,
-                    "text": "Down"
-                  },
-                  "1": {
-                    "color": "green",
-                    "index": 0,
-                    "text": "Up"
-                  }
+      "description": "Indicates if the database is up or not",
+      "fieldConfig": {
+        "defaults": {
+          "mappings": [
+            {
+              "options": {
+                "0": {
+                  "color": "red",
+                  "index": 1,
+                  "text": "Down"
                 },
-                "type": "value"
-              }
-            ],
-            "thresholds": {
-              "mode": "absolute",
-              "steps": [
-                {
+                "1": {
                   "color": "green",
-                  "value": null
-                },
-                {
-                  "color": "red",
-                  "value": 80
+                  "index": 0,
+                  "text": "Up"
                 }
-              ]
-            },
-            "unit": "short"
+              },
+              "type": "value"
+            }
+          ],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
           },
-          "overrides": []
+          "unit": "short"
         },
-        "gridPos": {
-          "h": 6,
-          "w": 2,
-          "x": 0,
-          "y": 5
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 6,
+        "w": 2,
+        "x": 0,
+        "y": 5
+      },
+      "hideTimeOverride": true,
+      "id": 4,
+      "links": [],
+      "options": {
+        "colorMode": "background",
+        "graphMode": "none",
+        "justifyMode": "auto",
+        "orientation": "auto",
+        "reduceOptions": {
+          "calcs": ["lastNotNull"],
+          "fields": "",
+          "values": false
         },
-        "hideTimeOverride": true,
-        "id": 4,
-        "links": [],
-        "options": {
-          "colorMode": "background",
-          "graphMode": "none",
-          "justifyMode": "auto",
-          "orientation": "auto",
-          "reduceOptions": {
-            "calcs": [
```

Comment:
@aangelisc I see that this PR https://github.com/grafana/grafana/pull/74953 got merged yesterday. When I merged `main` the linter fixed bunch of extra spaces in this json file. I went thought all of the changes ( I recommend turning on `Hide whitespace` in this PR) an only whitespace/space/newlines got removed based on our linter.

Reply:
Hey @ivanahuckova, thanks for this 😊 it was an external PR so the linter config may have been invalid. Thank you for fixing!